tests/e2e/tools/FFI/memory/memory_eat.c

```diff
@@ -12,7 +16,7 @@ int main() {
     }
 
     unsigned long total_memory = si.totalram * si.mem_unit;
-    unsigned long memory_to_allocate = 0.2 * total_memory;
+    unsigned long memory_to_allocate = MEM_PERCENT * total_memory;
```

Comment:
Consider here
Adding main arg parameter, that could be parsed through container env

Reply:
Not sure I fully understand. Do you want this to be a command line parameter, so that it can be set when starting the test binary? Currently it results in the same two binaries that there were before.

Reply:
If I understood, he is looking to get the value from env variable instead.

Reply:
This would be a behavioral change in the way the test is executed. If you think that makes sense then I would do it in a separate PR.

Reply:
In 2nd thought, I think only argparse is needed with default value
of MEM_PERCENT=50 This way the program could be called as follow:

podman run --rm  --env VARIABLE1=foobar -it quay.io/centos-sig-automotive/ffi-tools:latest bash -c "./QM/90_percent_memory_eat MEM_PERCENT=30"

Reply:
This implies not only having the same source file, but also having just a single identical binary.

If you want to be able to set the percentage via command line argument, then  can add this in a minute in a second PR.

Are there any concerns with merging the sources?

Reply:
SGTM
Any way we need to move with this:
#280 in order to run ffi tests in CI, and then merge current PR
PTAL # review #280

Reply:
> This implies not only having the same source file, but also having just a single identical binary.
> 
> If you want to be able to set the percentage via command line argument, then can add this in a minute in a second PR.
> 
> Are there any concerns with merging the sources?

@dennisbrendel I think you need to update also ffi tests, which call the binaries inside ffi-tools container